src/testinput/java/java7/util7/Collections.java

```diff
@@ -590,7 +591,7 @@ private static void swap(Object[] arr, int i, int j) {
             throw new IndexOutOfBoundsException("Source does not fit in dest");
 
         if (srcSize < COPY_THRESHOLD ||
-            (src instanceof RandomAccess && dest instanceof RandomAccess)) {
```

Comment:
What is the reason for the many whitespace changes in this file?
We should keep this file in sync with the actual JDK, to make comparisons easier in the future.
Does the new or the old indentation correspond to the JDK?